ext/B/B.pm

```diff
@@ -593,6 +593,18 @@ C<REFCNT> (corresponding to the C function C<SvREFCNT>).
 
 =item FLAGS
 
+=item is_bool
+
+Returns true if the SV is a boolean (PL_Yes or PL_No).
+
+=item is_bool_yes
+
+Returns true if the SV is a boolean and is PL_Yes.
+
+=item is_bool_no
+
+Returns true if the SV is a boolean and is PL_No.
+
```

Comment:
The code doesn't check whether the SV is PL_Yes or PL_No, it checks whether the SV is a boolean (which any SV can be) and for the second two, whether that value is true or false.

Reply:
going to adjust the description to match the suggestion for is_bool_yes